content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -2999,12 +2999,12 @@ A nested `disconnect` query breaks the connection between a parent record and a
 - `disconnect` is only available if the relation is optional.
 - If the relationship you are attempting to disconnect does not exist:
 
-	- ([In 2.21.0 and later](https://github.com/prisma/prisma/releases/tag/2.21.0)), the operation does nothing
```

Comment:
investigating this…

Reply:
Looks ok, it  replace `tab`s with spaces

<img width="348" alt="image" src="https://user-images.githubusercontent.com/4734297/124758941-8a8bf700-df48-11eb-9e28-6adc28e53d08.png">

<img width="728" alt="image" src="https://user-images.githubusercontent.com/4734297/124759006-9b3c6d00-df48-11eb-9fc1-a0b9f79e7bc5.png">